pdr_backend/pdr_dashboard/predictoor_dash.py

```diff
@@ -38,8 +41,8 @@ def predictoor_dash(ppss: PPSS, debug_mode: bool):
             Full error: {e}"""
         )
         return
-
-    get_callbacks(app)
+    get_callbacks_home(app)
```

Comment:
having a Page-like OOP structure would also help in these cases here. Maybe we can have callbacks autoregistered by each page instead of calling these functions. This comment you can ignore, if you think it's too "science-fiction" :)